shell/plugins/dashboard-store/actions.js

```diff
@@ -184,6 +186,13 @@ export default {
 
     console.log(`Find All: [${ ctx.state.config.namespace }] ${ type }`); // eslint-disable-line no-console
     opt = opt || {};
+    // we should always exclude managed fields when retreiving resources to cut down on payload size and store memory
+    if (isArray(opt?.excludeFields)) {
```

Comment:
There's two parts to the comment

1) Steve specific code should be in the steve store and not dashboard store (still pending)
  - The boundary is the files. steve actions inherit and in places override generic dashboard actions
2) The steve store caters for both steve, norman and kube api's
  - The boundary doesn't currently exist. 
  - Checking for v1 might work, but it would need to be from the start of the string (kube api contains v1)

Reply:
Given that other APIs don't seem to error out when passing in this parameter even if it's not used, I think we're safe with a simple '/v1/' check on the string unless you've got a simple bit of code you think would work well here.

The right solution would be to refactor the code so that the steve plugin (there isn't actually a steve store) is only used for code that pertains to the steve API but I would say that falls outside the scope of this issue.

Reply:
updated